### DISCUSSION TITLE
Mattermost Clubhouse

### DISCUSSION THREAD

**Discussion by hahmadia** [mattermost/community#147] [Practical Ideas]
With the new release of Discord Stages, allowing people to speak and select from the audience to ask question and such (a in house club house implementation), I think it be cool to implement a similar feature in Mattermost after having the structure and implementation down for voice channels.

This will be another step towards eliminating the use of third party applications for discussions and Q AND A!

**Comment by esethna** [comment]
Totally, could love to see this next level collaboration in Mattermost!
